### PR TITLE
#559 instance name for logging

### PR DESCRIPTION
**Pull request by evilchia**
Label is for logging reporting purposes so held in the LoggingService.  Overridable by the developer through property declaration.

### CODE REVIEW

File: src/component/ComponentInternalsImpl.ts [hunk 1]
```diff
@@ -595,7 +595,7 @@ class ComponentInternalsImpl implements ComponentInternals, Tellable {
 
 	private initFields(): void {
 		this.id = IdGenerator.INSTANCE.generate();
-		this.logger = LoggerFactory.getLogger(`${this.getName()} component ${this.id}`);
+		this.logger = LoggerFactory.getLogger(`Component<${this.getName()}> ${this.id}`);
```
Comment (matthewwerny) [inline src/component/ComponentInternalsImpl.ts:598]: Doesn't this make it look like it is a generic type?  I think that the square brackets that you used in ModuleImpl might be less confusing and more consistent.

**Review by matthewwerny — CHANGES_REQUESTED** [review]
Check the brackets comment

File: src/component/ComponentInternalsImpl.ts (hunk 1 above)
Comment (evilchia) [reply src/component/ComponentInternalsImpl.ts:598]: From among the collection of Modules the selected is logging, whereas the Component of type X is saying something, but for consistency, I have changed it.

**Review by matthewwerny — APPROVED** [review]
Looks good

**Pull request merged**